.github/workflows/conda.yml

```diff
@@ -57,3 +56,12 @@ jobs:
           mamba install anaconda-client
 
           anaconda upload --label dev noarch/*.tar.bz2
+      - name: Trigger distributed build
+        uses: peter-evans/repository-dispatch@v1.1.3
+        if: |
+          github.event_name == 'push'
+          && github.repository == 'dask/dask'
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
```

Comment:
The token required here is a `public_repo` scoped personal access token generated by a **user** with write access to Distributed.

Is there any "bot" user accounts associated with Dask that we could safely use for this?

Reply:
Yes! There is @dask-bot that is for this. @jrbourbeau probably knows how to log in as it to make a token.